addons/sale_crm/migrations/10.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -2,7 +2,8 @@
 sale_crm     / account.invoice          / campaign_id (many2one)        : NEW relation: utm.campaign
 sale_crm     / account.invoice          / medium_id (many2one)          : NEW relation: utm.medium
 sale_crm     / account.invoice          / source_id (many2one)          : NEW relation: utm.source
-# TODO: Try to migrate from sales order
+# NOTHING TO DO. Not required.
```

Comment:
As you have a field `invoice_ids` in `sale.order`, you should fill these fields from sale.order to the associated invoices.